.github/workflows/pypi-release.yaml

```
@@ -80,7 +80,7 @@ jobs:
           python -m pip install dist/xbatcher*.whl
           python -m xbatcher.util.print_versions
       - name: Publish package to TestPyPI
-        uses: pypa/gh-action-pypi-publish@v1.8.1
+        uses: pypa/gh-action-pypi-publish@v1.8.3
         with:
           password: ${{ secrets.TEST_PYPI_API_TOKEN }}
           repository_url: https://test.pypi.org/legacy/
```

Silence deprecation warning `Input 'repository_url' has been deprecated with message: The inputs have been normalized to use kebab-case. Use `repository-url` instead.`

```suggestion
          repository-url: https://test.pypi.org/legacy/
```